src/org/zaproxy/zap/extension/script/ScriptAPI.java

```diff
@@ -195,7 +195,10 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
 				throw new ApiException(ApiException.Type.DOES_NOT_EXIST, ACTION_PARAM_SCRIPT_NAME);
 			}
 			if (!script.getType().getName().equals(ExtensionScript.TYPE_STANDALONE)) {
-				throw new ApiException(ApiException.Type.ILLEGAL_PARAMETER, ACTION_PARAM_SCRIPT_NAME);
+				throw new ApiException(
+						ApiException.Type.ILLEGAL_PARAMETER,
+						"Parameter " + ACTION_PARAM_SCRIPT_NAME + " does not select a " + ExtensionScript.TYPE_STANDALONE

```

Comment:
Would “is not a” or “does not match a” make more sense.

Reply:
Yeah, changed.